docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -46,11 +47,18 @@ Follow these steps to "always trust" the Kyma certificate on macOS:
   cd installation
   ```
 
-2. Run the `run.sh` script:
+2. (For macOS users) Run the `run.sh` script:
   ```
   cmd/run.sh
   ```
 
+ (For Linux users) First start a minikube cluster, and run the `run.sh` script, because the hyperkit driver is not available on Linux:
+
+  ```
+  minikube start
+  cmd/run.sh --skip-minikube-start
+  ```
```

Comment:
How about improving `cmd/run.sh` script? 

I mean, we can do something like that:

```bash
if [ `uname` = "Linux" ]; then
  minikube start
  SKIP_MINIKUBE_START=true
fi
```

in the beginning of the script.
After that it should work seamlessly both on macOS and Linux out of the box.